server/controllers/exporters/xlsx.js

```diff
@@ -117,24 +118,28 @@ const buildXlsxSheet = function (workbook) {
             }
             break;
         case "networkOptimizedWeights":
-            if (Object.keys(workbook.networkOptimizedWeights).length > 0) {
-                resultSheet.push(
-                    {
-                        "name": "network_optimized_weights",
-                        "data": buildNetworkSheet(workbook.networkOptimizedWeights.genes,
-                            workbook.networkOptimizedWeights.links)
-                    }
-                );
+            if (exportNetworkType === "weighted") {
+                if (Object.keys(workbook.networkOptimizedWeights).length > 0) {
+                    resultSheet.push(
+                        {
+                            "name": "network_optimized_weights",
+                            "data": buildNetworkSheet(workbook.networkOptimizedWeights.genes,
```

Comment:
just small formatting stuff for readability. I would indent as so 
"data": buildNetworkSheet(
        workbook...,
        workbook...
)